baus/variables.py

```diff
@@ -147,19 +147,21 @@ def vacant_res_units(buildings, households):
 
 
 @orca.column('buildings', cache=True)
-def sqft_per_job(buildings, building_sqft_per_job, superdistricts,
-                 taz_geography, year):
-    sqft_per_job = buildings.\
-        building_type.fillna("O").map(building_sqft_per_job)
-
-    # this factor changes all sqft per job according to which superdistrict
-    # the building is in - this is so denser areas can have lower sqft
-    # per job - this is a simple multiply so a number 1.1 increases the
-    # sqft per job by 10% and .9 decreases it by 10%
-    superdistrict = misc.reindex(
-        taz_geography.superdistrict, buildings.zone_id)
-    sqft_per_job = sqft_per_job * \
-        superdistrict.map(superdistricts['sqft_per_job_factor_{}'.format(year)])
+def sqft_per_job(buildings, building_sqft_per_job, sqft_per_job_adjusters, telecommute_sqft_per_job_adjusters, taz_geography, base_year, year, run_setup):
+    
+    sqft_per_job = buildings.building_type.fillna("O").map(building_sqft_per_job)
+
+    superdistrict = misc.reindex(taz_geography.superdistrict, buildings.zone_id)
+
+    # this factor changes all sqft per job according to which superdistrict the building is in - this is so denser areas can have lower sqft per job
+    # this is a simple multiply so a number 1.1 increases the sqft per job by 10% and .9 decreases it by 10%
+
+    # if adjusters are enabled, adjust sqft_per_job rates for all years
+    if run_setup["run_telecommute_strategy"] and year != base_year:
+        sqft_per_job = sqft_per_job * superdistrict.map(telecommute_sqft_per_job_adjusters['sqft_per_job_factor_{}'.format(year)])
+	# if the telecommute strategy is enabled, instead adjust future year sqft_per_job rates with the factor for that year
```

Comment:
(Substantive question)

Reply:
@akselx I think since it's `elif` it works out. So if telecommute is enabled, it will apply the yearly rates. If it's not, it will only apply the adjusters, if they are enabled. If both are true, it will apply the telecommute rates, but won't move on to apply the `elif` statement. 

Ultimately, we really want the two sources of adjusters to be additive, so that we can see what each is in it of itself. But currently, the telecommute rates actually already have the adjusters factored into them (as far as I can surmise), so I set it up this way. In this PR, I created two files- one for telecommute and one for adjusters (calibration of some sort). I figured that because the values themselves are hard to tease out, that would have to happen when we're willing for the results to change.

Reply:
Right - the `elif` will apply the first one to be true so if telecommuting is set, that will be the one that goes. I think you are saying that telecommuting adjustments already bake in the baseline adjustment, so the additive part is handled on the data side.